R/write_ts.R

```diff
@@ -0,0 +1,21 @@
+
+#'@title write timeseries format compressed file
+#'@param data unitted data.frame
+#'@param site the full site name (e.g., 'nwis_06893820')
+#'@param variable the variable name of the output
+#'@param folder the folder to write the file in
+#'@keywords internal
+#'@export
+write_ts <- function(data, site, variable, folder){
+  
```

Comment:
would it make sense to do some error checking here? we could check that:
- data has two columns
- first column as "UTC" units
- second column has same name as variable (or ts_name, whichever it is)

These are already handled by stage_nwis_ts and stage_nldas_ts, but it'd be nice to have the error checks in case there are additional stage_xx_ts functions still to be written.